app/views/posts/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="a-post">
+<div class="post-id ">
+  <div class="post-info">
+      <h4>Post <%= @post.id %> by <%= @post.user.name %></h4>
+      <div class="likes-comment">
+        <p class="comment">comments: <%= @post.comments.count %></p>
+        <p class="likes"> likes: <%= @post.likes.count %></p>
+      </div>
+    </div>
+    <p class="post-text"> <%= @post.text %></p>
+  </div>
+  <div class="all-comments">
+    <% @post.comments.each do |comment| %>
+        <strong><%= comment.user.name %></strong>: <%= comment.text %>
+    <% end %>
+  </div> 
+</div>
+
+</div>
```

Comment:
- [ ] Kindly place the elements correctly.

  Your Implementation | Wireframe
  :---:|:---:
  ![image](https://github.com/hydaline-code/myblogapp/assets/36542986/cbdf8733-d4b8-4da0-838f-441ac8ad8952) | ![image](https://github.com/hydaline-code/myblogapp/assets/36542986/1b16067b-28ce-4a9d-862e-408a2467b60a)